### PR TITLE
Remove 3x DS multiplier and IT products for DynamicScale from public facing dev docs

### PR DESCRIPTION
**Pull request by prasanthmathialagan-okta**
## Description:
- **What's changed?** 
Changed `3x to 1000x` to `5x to 1000x`.
Changed `e.g. One App, Enterprise or IT Products` to `One App or Enterprise`

- **Is this PR related to a Monolith release?** 
No

### Resolves:

* [OKTA-326593](https://oktainc.atlassian.net/browse/OKTA-326593)

### CODE REVIEW

**Comment by jakubvul-okta** [comment]
I will wait on @karthicksolaimalai-okta 's approval for this one.

**Comment by karthicksolaimalai-okta** [comment]
Looks good to me! Thanks Prasanth.

**Review by karthicksolaimalai-okta — APPROVED** [review]
lgtm - thanks Prasanth.

**Review by jakubvul-okta — APPROVED** [review]

**Pull request merged**
